File: Java/Maths/Multiplication table.java
import java.util.Scanner;
class multiplicationTable {
    public static void main(String args[]){
        Scanner input = new Scanner(System.in);
        System.out.print("Enter a positive number: ");
        int num = input.nextInt();
        System.out.println("\nMultiplication table of: " + num);
        for (int i = 1; i <= 10; i++) {
            System.out.println(num + " x " + i + " = " + (num*i));
        }
    }
}

File: C++/Maths/Multiplication table.cpp
#include <iostream>
using namespace std;

int main(){
    int num;
    cout << "Enter a positive number: ";
    cin >> num;
    cout << "\nMultiplication table of: " << num << endl;
    for (int i = 1; i <= 10; i++) {
        cout << num << " x " << i << " = " << (num*i) << endl;
    }
}